webpack.config.js

```diff
@@ -144,41 +146,33 @@ const mainConfig = {
   },
 };
 
-const indexConfig = {
-  entry: "./src/loadPanel.js",
-  mode: "production",
-  output: {
-    path: join(__dirname, "dist"),
-    filename: "loadPanel.js",
-    publicPath: "./",
-  },
-  module: {
-    rules:
-    [{
-      test: /\.(png|jpe?g|gif)$/i,
-      use: [
+const manifest = {
+    devtool: "inline-cheap-module-source-map",
+    mode: "development",
+    entry: "./src/manifest.json",
+    output: {
+      path: join(__dirname, "dist"),
+      publicPath: "./",
+      assetModuleFilename: '[name][ext]',
```

Comment:
Can you pls replace '' with "" here as well?

Reply:
sure